VideoLocker/src/main/java/org/edx/mobile/user/UserAPI.java

```diff
@@ -0,0 +1,44 @@
+package org.edx.mobile.user;
+
+import android.support.annotation.NonNull;
+
+import com.google.gson.FieldNamingPolicy;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import com.squareup.okhttp.OkHttpClient;
+
+import org.edx.mobile.discussion.RetroHttpExceptionHandler;
+import org.edx.mobile.http.RetroHttpException;
+import org.edx.mobile.util.Config;
+
+import retrofit.RestAdapter;
+import retrofit.client.OkClient;
+import retrofit.converter.GsonConverter;
+
+@Singleton
+public class UserAPI {
+
+    private UserService userService;
+
+    @Inject
+    public UserAPI(@NonNull Config config, @NonNull OkHttpClient client) {
+        Gson gson = new GsonBuilder()
+                .setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
+                .setDateFormat("yyyy-MM-dd'T'HH:mm:ssZ").create();
```

Comment:
I was wondering that, but I don't know if different endpoints have different rules regarding field name, date formatting, etc.